Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIPermissionsCocoa.mm

```diff
@@ -70,12 +72,73 @@
     HashSet<Ref<WebExtensionMatchPattern>> matchPatterns;
     parseMatchPatterns(origins, matchPatterns);
 
+    // If there is nothing to grant, return YES. This matches Chrome and Firefox.
+    if (!permissions.size() && !origins.size()) {
+        // FIXME: <https://webkit.org/b/250238> Invoke onAdded listeners.
+        completionHandler(true);
+        return;
+    }
+
     if (hasPermissions(permissions, matchPatterns)) {
         completionHandler(true);
         return;
     }
 
-    // FIXME: <https://webkit.org/b/250135> Call delegate method to prompt user for access.
+    // There shouldn't be any unsupported permissions, since they got reported as an error before this.
+    ASSERT(!permissions.isSubset(extension().supportedPermissions()));
```

Comment:
Should this be: `ASSERT(permissions.isSubset(extension().supportedPermissions()));`?

Reply:
Yes! I'll fix this.